### ISSUE TITLE
Feature request: will you open the API about read/hit count in article meta info?

### ISSUE THREAD

**Issue by askender** [thematters/developer-resource#4]
RT

some articles about it:
<https://matters.news/@gonghei88/%E6%AA%A2%E8%A8%8E%E6%88%91%E7%9A%84-%E9%80%8F%E9%81%8Elikecoin%E5%92%8Cmatters%E6%8E%A8%E8%A1%8C%E6%8F%9B%E6%9B%B8-%E9%80%81%E6%9B%B8-%E8%A8%88%E5%8A%83-bafyreihkxgv4ijjd6jer2ywysfpic3fzqpnwrk5nxdes6c4n6vysogcpoa>

**Comment by guoliu** [comment]
@askender Are there UI designs you have in mind? If it's available not only on API but also in product, it will be useful to more people.

**Comment by askender** [comment]
@guoliu   Yes. I have a plan of Matters DApp (likecoin is used too) website.  UI design is included.

Besides, it is also for Datters (a project of matters, data analysis is needed).
And a part of the DApp is also a project in HackFS.

**Comment by guoliu** [comment]
@askender sounds good, adding this to TODO list, and we will try to pick it up as soon as possible. Off course you are more than welcome to offer a hand 😄
